log4j-perf/src/main/java/org/apache/logging/log4j/perf/jmh/StringEncodingBenchmark.java

```diff
@@ -52,11 +53,11 @@
     private static final String STRING_ISO8859_1 = "ISO-8859-1";
     private static final String STRING_US_ASCII = "US-ASCII";
```

Comment:
Would you mind replacing these too, please? That is, `StandardCharsets.ISO_8859_1.name()`. Otherwise the new usage seems inconsistent to me.